.github/workflows/mtags-auto-release.yml

```diff
@@ -8,11 +8,11 @@ on:
       metals_version:
         description: "Metals Version"
         required: true
-        default: "v0.11.5"
+        default: "v0.11.6"
       metals_ref:
         description: "Tag/branch-name from which run release"
         required: true
-        default: "v0.11.5_mtags_release"
+        default: "v0.11.6_mtags_release"
```

Comment:
you need either to put `v0.11.6` or create a branch `v0.11.6_mtags_release` from v0.11.6 tag.
I would prefer to update it to `v0.11.6` after release.
If there will be a need to port some changes for mtags release we could switch it on branch at any time

Reply:
Ach, right, fixed to just `v0.11.6`, thank you for pointing out!